gradle.properties

```diff
@@ -0,0 +1,14 @@
+GROUP=com.apollostack.android
+VERSION_NAME=0.0.1-SNAPSHOT
+
+POM_URL=https://github.com/apollostack/apollo-android/
+POM_SCM_URL=https://github.com/square/apollo-android/
```

Comment:
Should this be 'apollostack' instead of 'square' in the GitHub URL? (also the two lines down)